<a href="https://colab.research.google.com/github/purvanshgarg/outlet_prediction/blob/main/outlet_prediction.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [3]:
import numpy as np
import pandas as pd
data= pd.read_csv("/content/Foodtruck.csv")
data.head()

Population   Profit
0      6.1101  17.5920
1      5.5277   9.1302
2      8.5186  13.6620
3      7.0032  11.8540
4      5.8598   6.8233

In [14]:
x=data["Population"]
y=data["Profit"]
x
y

0     17.59200
1      9.13020
2     13.66200
3     11.85400
4      6.82330
        ...   
92     7.20290
93     1.98690
94     0.14454
95     9.05510
96     0.61705
Name: Profit, Length: 97, dtype: float64

In [19]:
from sklearn.linear_model import LinearRegression
model=LinearRegression()
X=np.array(x).reshape(-1,1)
Y=np.array(y).reshape(-1,1)
model.fit(X,Y)

LinearRegression(copy_X=True, fit_intercept=True, n_jobs=None, normalize=False)

In [24]:
pred=model.predict(np.array(3.703).reshape(1,-1))
print(pred)

[[0.52202271]]


#estimated value of profit if I set up an outlet in jaipur having 3.073 million population is 0.52202271 so as this brings a profit so this is feasible 